### PR TITLE
Bugfix no payments from requestor not handled in the forced payment uc

### PR DESCRIPTION
**Pull request by Jakub89**
Resolves: #980

### CODE REVIEW

File: concent_api/core/tests/test_payments_bankster.py [hunk 1]
```diff
@@ -360,6 +360,44 @@ def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount
...
+            ]
+
+        with freeze_time("2018-02-05 10:00:25"):
+            with mock.patch('core.payments.bankster.service.get_deposit_value', return_value=100000) as get_deposit_value_mock:
+                with mock.patch(
+                    'core.payments.bankster.service.get_list_of_payments',
+                    side_effect=[
+                        self._get_empty_list_of_transactions(),
+                        self._get_empty_list_of_transactions(),
+                    ]
```
[the remark this answers is not in the document]
Comment (Jakub89) [reply concent_api/core/tests/test_payments_bankster.py]: Yes, once for `list_of_transactions` and once for `list_of_forced_payments` in bankster.

File: concent_api/core/payments/bankster.py [hunk 2]
```diff
@@ -367,23 +367,7 @@ def settle_overdue_acceptances(
...
-            # Concent checks if all passed SubtaskResultAccepted messages from subtask_results_accepted_list
-            # have payment_ts < T1
-            if any(
-                youngest_transaction_timestamp > subtask_results_accepted.payment_ts
-                for subtask_results_accepted in acceptances
-            ):
-                raise BanksterTimestampError
-        else:
-            raise BanksterTimestampError
+        validate_list_of_transactions_timestamps(list_of_transactions, cut_off_time, acceptances)
```
Comment (rwrzesien) [inline concent_api/core/payments/bankster.py]: So what about if `len(list_of_transactions) == 0` ?

Comment (Jakub89) [reply concent_api/core/payments/bankster.py]: Nothing, it pass validation and `settle_overdue_acceptances` return `claim_against_requestor`.

File: concent_api/core/tests/test_payments_bankster.py [hunk 3]
```diff
@@ -360,6 +360,44 @@ def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount
...
+                        acceptances=subtask_results_accepted_list,
+                        requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
+                    )
+
+        get_deposit_value_mock.assert_called_once()
+        get_list_of_payments_mock.assert_called()
+
+        self.assertIsNotNone(claim_against_requestor.tx_hash)
+
+        self.assertEqual(claim_against_requestor.amount, 15000)
```
[the remark this answers is not in the document]
Comment (Jakub89) [reply concent_api/core/tests/test_payments_bankster.py]: Done

**Pull request merged**